composer.json

```diff
@@ -31,7 +31,11 @@
   ],
   "scripts": {
     "test": "vendor/bin/phpunit -c tests/phpunit.xml --testdox --color=always",
-    "test:watch": "vendor/bin/phpunit-watcher watch -c tests/phpunit.xml --testdox"
+    "test:watch": "vendor/bin/phpunit-watcher watch -c tests/phpunit.xml --testdox",
+    "infection": "vendor/bin/infection --threads=4 --min-msi=80 --min-covered-msi=80 --ansi",
+    "phpstan": "vendor/bin/phpstan analyse -c phpstan.neon -l 2 src tests --autoload-file=vendor-bin/phpunit/vendor/autoload.php --ansi",
+    "post-install-cmd": ["@composer bin all install --ansi"],
```

Comment:
this should be in a seperate pr

Reply:
we need this.
If we don't have that, Dependency Hell opens up.